src/core/Akka/Util/MatchHandler/CachedMatchCompiler.cs

```diff
@@ -71,27 +71,6 @@ private Delegate CompileToDelegate(IReadOnlyList<TypeHandler> handlers, IReadOnl
             delegateArguments = result.Arguments;
             return compiledLambda;
         }
-
-#if !CORECLR
```

Comment:
Byterot, these codes were never compiled before the cleanup (since we moved to single `netstandard2.0` platform target)